tests/test_more.py

```diff
@@ -1112,6 +1113,55 @@ def test_seperators(self):
             self.comp_with_str_split(s, delim, maxsplit)
```

Comment:
Back on the earlier `keep_separator` PR, I should have asked this: your function does this:

```python
>>> from more_itertools import split_at
... 
... list(split_at('abccba', lambda x: x == 'a', keep_separator=True))
[[], ['a'], ['b', 'c', 'c', 'b'], ['a'], []]
```

Is that really what we want, the blanks at the ends?

Reply:
Sorry for the late answer. The closest example I found is:

```
>>> import re
>>> re.split("(a)", 'abccba')
['', 'a', 'bccb', 'a', '']
```

I think it's a strong argument in favour of the current behavior.